analyzer/QUIC.spicy

```diff
@@ -236,6 +252,25 @@ public type LongHeaderPacket = unit {
   src_conn_id: bytes &size=self.client_conn_id_length;
 
   switch ( self.version ) {
+    VersionDraft22,
+    VersionDraft23,
+    VersionDraft24,
+    VersionDraft25,
+    VersionDraft26,
+    VersionDraft27,
+    VersionDraft28,
+    VersionDraft29,
+    VersionDraft30,
+    VersionDraft31,
+    VersionDraft32,
+    VersionDraft33,
+    VersionDraft34 -> d: LongHeaderPacketV1(self);
+    VersionFace001,
+    VersionFace002,
+    VersionFace00e,
+    VersionFace011,
+    VersionFace012,
+    VersionFace013 -> d: LongHeaderPacketV1(self);
```

Comment:
This is identical to the previous case, was that on purpose? If intentional simply use a single case for all of these.